crates/ursa-network/src/service.rs

```diff
@@ -11,40 +11,46 @@
 //! The [`Swarm`] events are processed in the main event loop. This loop handles dispatching [`NetworkCommand`]'s and
 //! receiving [`UrsaEvent`]'s using the respective channels.
 
-use std::collections::{HashMap, HashSet};
-use std::num::{NonZeroU8, NonZeroUsize};
-use std::sync::Arc;
 use anyhow::{anyhow, Error, Result};
 use bytes::Bytes;
-
 use cid::Cid;
 use db::Store as Store_;
 use fnv::FnvHashMap;
 use futures_util::stream::StreamExt;
 use fvm_ipld_blockstore::Blockstore;
 use libipld::DefaultParams;
-use libp2p::autonat::{Event as AutonatEvent, NatStatus};
-use libp2p::gossipsub::error::{PublishError, SubscriptionError};
-use libp2p::gossipsub::{MessageId, TopicHash};
-use libp2p::identify::Event as IdentifyEvent;
-use libp2p::multiaddr::Protocol;
-use libp2p::ping::Event as PingEvent;
-use libp2p::request_response::{RequestResponseEvent, RequestResponseMessage};
-use libp2p::swarm::{ConnectionHandler, IntoConnectionHandler, NetworkBehaviour};
-use libp2p::Multiaddr;
 use libp2p::{
-    gossipsub::IdentTopic as Topic,
+    autonat::{Event as AutonatEvent, NatStatus},
```

Comment:
Was this automated or manually done?

Reply:
https://rust-lang.github.io/rustfmt/?version=v1.5.1&search=#imports_granularity

I have it set locally but we could easily define a rustfmt.toml and enforce this formatting for everyone's `cargo fmt`

Reply:
We don't currently enforce fmt, the default does a pretty good job and it's consistent across devs.